srcalc/src/test/java/gov/va/med/srcalc/web/view/admin/EditRuleTest.java

```diff
@@ -0,0 +1,53 @@
+package gov.va.med.srcalc.web.view.admin;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.mockito.Mockito.mock;
+import gov.va.med.srcalc.db.RiskModelDao;
+import gov.va.med.srcalc.db.RuleDao;
+import gov.va.med.srcalc.db.VariableDao;
+import gov.va.med.srcalc.domain.model.Rule;
+import gov.va.med.srcalc.service.DefaultAdminService;
+import gov.va.med.srcalc.service.InvalidIdentifierException;
+
+import java.util.ArrayList;
+
+import org.junit.Test;
+
+public class EditRuleTest
+{
+    private final DefaultAdminService fAdminService = new DefaultAdminService(
+            mock(VariableDao.class), mock(RiskModelDao.class), mock(RuleDao.class));
+    
+    @Test
+    public final void testBasic()
+    {
+        final EditRule editRule = new EditRule(fAdminService);
```

Comment:
If you accepted a `ModelInspectionService`, you can just use `MockModelService` here.